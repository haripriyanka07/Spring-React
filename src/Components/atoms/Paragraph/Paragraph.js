import { Typography } from "@material-ui/core";
import "./Paragraph.css";

const Paragraph = (props) => {
    return (
        <Typography align="center" paragraph className="paragraph">{props.children}</Typography>
    )
}
export default Paragraph;