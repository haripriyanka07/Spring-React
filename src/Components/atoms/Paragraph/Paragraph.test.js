import React from 'react';
import renderer from 'react-test-renderer';
import Paragraph from './Paragraph';

it('test the paragraph', () => {
    const renderedComponent = renderer.create(<Paragraph>This is paragraph testing</Paragraph>).toJSON();
    expect(renderedComponent).toMatchSnapshot();
})