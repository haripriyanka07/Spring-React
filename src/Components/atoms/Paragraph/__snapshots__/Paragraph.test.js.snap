// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test the paragraph 1`] = `
<p
  className="MuiTypography-root paragraph MuiTypography-body1 MuiTypography-paragraph MuiTypography-alignCenter"
>
  This is paragraph testing
</p>
`;
